graphprot/NeuralNet.py

```diff
@@ -95,29 +95,79 @@ def __init__(self, database, Net,
 
         elif self.task == 'class':
             self.loss = nn.CrossEntropyLoss(weight = self.class_weights, reduction='mean')       
+        
+        
+    def plot_loss(_train_loss, _valid_loss=[], nepoch):        
```

Comment:
I'm getting picky here but usually '_name' variable names are only used as internal. So not used as argument of functions.